src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -26,7 +35,7 @@
         return ok().body(
             AboutToStartOrSubmitCallbackResponse
                 .builder()
-                .errors(ImmutableList.of())
+                .errors(callbackProcessor.process(type,callback.getEventId(),callback.getCaseDetails()))
```

Comment:
Does checkstyle check kotlin files? (Missing spaces after `,`)
Something to remember to include in the Kotlin proposal...